File: BaekjoonAlgorithms/06-OneDimensionalArray/Python/2577.py
# 세 자연수 A x B x C를 계산한 결과에 0부터 9까지 각각의 숫자가 몇 번 쓰였는지 구하기

A = int(input())
B = int(input())
C = int(input())


def count_array(A, B, C):
    num = list(str(A * B * C))
    num_count = []
    for i in range(10):
        num_count.append(str(num.count(str(i))))

    return num_count


print("\n".join(count_array(A, B, C)))

File: BaekjoonAlgorithms/06-OneDimensionalArray/C/2577.c
/*세 개의 자연수를 곱했을 경우 0부터9까지의 개수*/
#include <stdio.h>

int main()
{
    int a,b,c;
    int x;
    char num[11]={0,};
    scanf("%d",&a);
    scanf("%d",&b);
    scanf("%d",&c);
    x=a*b*c;
    
	while(x>0) //1의자리숫자의 개수
	{
        num[x%10]++;
		x=x/10;
	}
    printf("%d\n",num[0]);
    printf("%d\n",num[1]);
    printf("%d\n",num[2]);
    printf("%d\n",num[3]);
    printf("%d\n",num[4]);
    printf("%d\n",num[5]);
    printf("%d\n",num[6]);
    printf("%d\n",num[7]);
    printf("%d\n",num[8]);
    printf("%d\n",num[9]);
}
